ui/react-components/components/AddAppointment/AddAppointment.test.jsx

```diff
@@ -2,10 +2,8 @@ import React from "react";
 import AddAppointment from "./AddAppointment.jsx";
 import {renderWithReactIntl} from "../../utils/TestUtil";
 import {fireEvent, waitForElement} from "@testing-library/react";
-import * as addAppointmentService from "./AddAppointmentService.js";
+import * as apiService from "./AddAppointmentService.js";
```

Comment:
any reason for renaming to a generic "apiService" if its just adding appt? I am ok, since this is a test

Reply:
Thanks for pointing it @angshu . The test file has been overridden when resolving merge conflicts hence the issue resolved the same. You can check now